spec/ciSmokeConf.js

```diff
@@ -39,7 +39,7 @@ exports.config = {
     'tunnel-identifier': process.env.TRAVIS_JOB_NUMBER,
     'build': process.env.TRAVIS_BUILD_NUMBER,
     'name': 'Protractor smoke tests',
-    'version': '9.1',
+    'version': '8',
     'selenium-version': '2.53.1'
```

Comment:
This should also be downgraded back to 2.44

```
-    'selenium-version': '2.44.0' // Use an old version because Safari has		 +    'selenium-version': '2.53.1'
 -                                 // issues loading pages after 2.44.
```